### PR TITLE
FEATURE: Allow also UI 4.*

### PR DESCRIPTION
**Pull request by jonnitto**

### CODE REVIEW

**Comment by breadlesscode** [comment]
Did you test it?

**Comment by pmaas** [comment]
Would be nice to merge this PR.

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "neos/neos-ui-compiled": "^3.0 | ^5.0"
+        "neos/neos-ui-compiled": "^3.0 | ^4.0 | ^5.0"
```
Comment (pmaas) [inline composer.json]: Shouldn't it be a `||`?
```
"neos/neos-ui-compiled": "^3.0 || ^4.0 || ^5.0"
```

Comment (breadlesscode) [reply composer.json]: Yes please update it to `||`. :) Thank you 👍

Comment (breadlesscode) [reply composer.json]: Wait i will fix it :)

**Comment by breadlesscode** [comment]
Merged, thank you all

**Pull request merged**